tasks/main.yml

```diff
@@ -113,7 +113,7 @@
   when:
     - network_provider == "initscripts"
 
-- name: Configure networking connection profiles
+- name: Configure networking connection profiles  # noqa jinja[invalid]
```

Comment:
What is the error here?  The jinja to me looks valid

Reply:
```
jinja[invalid]: An unhandled exception occurred while templating '{{ lookup('template', 'get_ansible_managed.j2') }}'. Error was a <class 'ansible.errors.AnsibleError'>, original message: An unhandled exception occurred while running the lookup plugin 'template'. Error was a <class 'ansible.errors.AnsibleError'>, original message: the template file get_ansible_managed.j2 could not be found for the lookup. the template file get_ansible_managed.j2 could not be found for the lookup
~/network_wl_ansible_lint/network/tasks/main.yml:116 Task/Handler: Configure networking connection profiles
```

Reply:
What version of ansible-lint are you using?  How are you running it?

Reply:
The version is ansible-lint 6.11.0. `ansible-lint -v`

Reply:
I upgraded the ansible-lint into 6.12.1, then I did not see the error.

Reply:
ok - then please remove the noqa comment